examples/md-flexible/src/configuration/objects/CubeUniform.h

```diff
@@ -73,20 +72,17 @@ class CubeUniform : public Object {
    * @param particles The container where the generated particles will be stored.
    */
   void generate(std::vector<ParticleType> &particles) const override {
-    ParticleType particle = getDummyParticle(particles.size());
+    // Wrapper so that std::vector can be used as an AutoPas::ParticleContainer
+    auto particlesWrapper = autopasTools::PseudoContainer(particles);
+
+    using namespace autopas::utils::ArrayMath::literals;
+    const auto boxMax = _bottomLeftCorner + _boxLength;
 
-    // Set up random number generation
-    std::random_device randomDevice;
```

Comment:
This is less an issue with the PR, but just something I noted:
* The legacy version here used the C++ style of generating random numbers
* The now employed `RandomGenerator` uses the C-style `rand()` function.

This could be resolved by:
- [ ] Adapt `double autopasTools::generators::RandomGenerator::fRand(double fMin, double fMax)` to also use the `std::uniform_distribution` and that the function takes a seed
- [ ] Remove the `srand()` calls from the `fillWithParticles` methods

It is not necessary for this PR, but it could be flagged as `TODO` at least. So, feel free to disregard it.